website/docs/r/resourcemanager_lien.html.markdown

```diff
@@ -0,0 +1,105 @@
+#
+# ----------------------------------------------------------------------------
+#
+#     This file is automatically generated by Magic Modules and manual
+#     changes will be clobbered when the file is regenerated.
+#
+#     Please read more about how to change this file in
+#     .github/CONTRIBUTING.md.
+#
+# ----------------------------------------------------------------------------
+layout: "google"
+page_title: "Google: google_resourcemanager_lien"
+sidebar_current: "docs-google-resourcemanager-lien"
+description: |-
+  A Lien represents an encumbrance on the actions that can be performed on a resource.
+---
+
+# google\_resourcemanager\_lien
+
+A Lien represents an encumbrance on the actions that can be performed on a resource.
+
+
+## Example Usage
+
+```hcl
+resource "random_id" "r" {
+  byte_length = 8
+}
+
+resource "google_project" "project" {
+  project_id = "project-${random_id.r.hex}"
+  name = "A very important project!"
+}
+
+resource "google_resourcemanager_lien" "lien" {
+  parent = "projects/${google_project.project.number}"
+  restrictions = ["resourcemanager.projects.delete"]
+  origin = "machine-readable-explanation"
+  reason = "This project is very important to me!"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+
+
+- - -
+
+* `reason` -
+  (Optional)
+  Concise user-visible strings indicating why an action cannot be performed
+  on a resource. Maximum lenth of 200 characters.
+* `origin` -
+  (Optional)
+  A stable, user-visible/meaningful string identifying the origin
+  of the Lien, intended to be inspected programmatically. Maximum length of
+  200 characters.
+* `parent` -
+  (Optional)
+  A reference to the resource this Lien is attached to.
+  The server will validate the parent against those for which Liens are supported.
+  Since a variety of objects can have Liens against them, you must provide the type
+  prefix (e.g. "projects/my-project-name").
+* `restrictions` -
+  (Optional)
+  The types of operations which should be blocked as a result of this Lien.
+  Each value should correspond to an IAM permission. The server will validate
+  the permissions against those for which Liens are supported.  An empty
+  list is meaningless and will be rejected.
+  e.g. ['resourcemanager.projects.delete']
+
+
+## Attributes Reference
+
+In addition to the arguments listed above, the following computed attributes are exported:
+
+* `name` -
+  A system-generated unique identifier for this Lien.
+* `create_time` -
+  Time of creation
+
+
+## Timeouts
+
+This resource provides the following
+[Timeouts](/docs/configuration/resources.html#timeouts) configuration options:
+
+- `create` - Default is 4 minutes.
+- `delete` - Default is 4 minutes.
+
+## Import
+
+Lien can be imported using any of these accepted formats:
+
+```
+$ terraform import google_resourcemanager_lien.default liens/{{name}}
+$ terraform import google_resourcemanager_lien.default {{name}}
```

Comment:
Hum, why do we generate twice the same format?

Reply:
Because the second one removes `{{project}}` but that's not in the import ID here.  I'm going to fix that.